src/styles/style.scss

```diff
@@ -95,3 +95,13 @@ label {
   box-shadow: inset 0 0 2px 0 rgba(159, 194, 0, 0.85);
   background: black;
 }
+
+@keyframes p-progress-spinner-color {
+  100%,
+  0% {
+      stroke: hsl(0,0%,98.04%);
+  }
+  100% {
+      stroke: hsl(0,0%,98.04%);
+  }
+}
```

Comment:
Why do you set this style rules here, but not in the spinner?

Reply:
I was trying add them scoped in component it self, but they doesn't apply this way... Some weird behavior of primeview styling.